connectors/sources/atlassian.py

```diff
@@ -17,6 +17,11 @@
 RETRIES = 3
 RETRY_INTERVAL = 2
 USER_BATCH = 50
+DATA_CENTER_USER_BATCH = 1000
+JIRA_CLOUD = "jira_cloud"
+JIRA_SERVER = "jira_server"
+CONFLUENCE_CLOUD = "confluence_cloud"
+CONFLUENCE_SERVER = "confluence_server"
```

Comment:
Same as above; can we remove these from confluence.py and jira.py and use these as imports? I think having a single source is better.

Reply:
I think there will be a circular import in that case. we can't import partially initiated jira.py. 
However, we can import it inside the method itself to avoid circular import error.